scopes/dependencies/dependencies/dependencies-cmd.ts

```diff
@@ -306,6 +315,14 @@ export class DependenciesUsageCmd implements Command {
   }
 }
 
+function isComponentId(depName: string) {
```

Comment:
This code should be moved to the main runtime

Reply:
isn't this just a utility function that doesn't need to be aware of any context? What benefit does it have from being part of the runtime class?

Reply:
I moved the function to main runtime file.